apps/frontend/src/pages/projects/libs/components/project-card/project-card.tsx

```diff
@@ -31,11 +37,33 @@ const ProjectCard = ({
 		onDelete(project);
 	}, [onDelete, project]);
 
+	const currentDate = new Date();
+	const lastActivityDate = project.lastActivityDate
+		? new Date(project.lastActivityDate)
+		: null;
```

Comment:
should I create `startOfDay` helper (just wrapper around existing startOfDay method in `date-fns`) for setting date `Wed Sep 18 2024 00:00:00`

Reply:
Do we need it with the current approach? I don't see where it's currently used

Reply:
just checking, maybe we still don't want hours/minutes/seconds be passed and prefer  `... 00:00:00`

Reply:
If it works as it is, then we don't